apps/site/src/app/(home)/sections/Countdown/Countdown.module.scss

```diff
@@ -0,0 +1,127 @@
+.paraboloid {
+	position: relative;
+	width: 100%;
+}
+
+.countdownWrapper {
+	display: flex;
+	align-items: center;
+	flex-direction: column;
+	position: relative;
+}
+
+.descriptiveText {
```

Comment:
Nitpick: Because this class is used on a div, maybe change the name to `description-container` or something.

Also, I don't know if camel case class names is standard or having dashes in between words is standard or if there even is a standard.